File: android-uitests/testData/CodeGeneration/app/src/main/java/com/codegeneration/MainActivity.java
package com.codegeneration;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        String hello = "hello";
        addNewCar();
        new Inventory().addNewCar();

    }

    private void addNewCar() {
        Car car = Car.getInstance();
        car.setMake("BMW");
        car.setModel("M5");
        car.setYear(2019);

        String make = car.getMake();
        String model = car.getModel();
        int year = car.getYear();

    }
}

File: android/testData/projects/multipleModuleDependOnAar/app/src/main/java/com/example/google/androidx/MainActivity.kt
package com.example.google.androidx

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity;

import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val color = androidx.appcompat.R.color.abc_tint_default
    }
}
